src/puppet.spec.ts

```diff
@@ -69,6 +69,9 @@ class PuppetTest extends Puppet {
   public async contactList ()                    : Promise<string[]> { return {} as any }
   public async contactQrcode (contactId: string) : Promise<string> { return { contactId } as any }
 
+  public async contactSelfName (newName: string) : Promise<boolean> { return true }
```

Comment:
We should follow the current naming style, like we used `contactQrcode` instead of `contactSelfQrcode`

Reply:
I would argue on this issue. Since getting qrcode, modifying name or signature, these operations are only allowed to be done on `ContactSelf`, so it makes sense to have `self` inside of the function name to indicate that this function is only related to self, not every contact. Does this make sense to you?

Reply:
If they are __ONLY__ allowed to be done on Self, why we need to point this out again?

Reply:
When we do coding ourselves, or some other contributors who do not familiar with the logic in `wechaty`, people will make mistake on that, so name it more specifically could avoid unnecessary mistake.